pyqtorch/circuit.py

```diff
@@ -58,8 +66,77 @@ def run(self, state: State = None, values: dict[str, Tensor] | ParameterDict = {
         return self.forward(state, values)
 
     def forward(self, state: State, values: dict[str, Tensor] | ParameterDict = {}) -> State:
+        if self.dropout_mode == DropoutMode.ROTATIONAL:
```

Comment:
can we move this into the init of QuantumCircuit? similar to https://github.com/pasqal-io/pyqtorch/blob/main/pyqtorch/parametric.py#L31